packages/app/src/airtaskerTechRadarData.ts

```diff
@@ -0,0 +1,385 @@
+import { RadarQuadrant, RadarRing, RadarEntry } from '@backstage/plugin-tech-radar';
+
+const quadrants = new Array<RadarQuadrant>();
+quadrants.push({ id: 'infrastructure', name: 'Infrastructure' });
+quadrants.push({ id: 'frameworks', name: 'Frameworks' });
+quadrants.push({ id: 'languages', name: 'Languages' });
+quadrants.push({ id: 'data', name: 'Data Management' });
+
+const rings = new Array<RadarRing>();
+rings.push({ id: 'use', name: 'USE', color: '#93c47d' });
+rings.push({ id: 'trial', name: 'TRIAL', color: '#93d2c2' });
+rings.push({ id: 'assess', name: 'ASSESS', color: '#fbdb84' });
+rings.push({ id: 'hold', name: 'HOLD', color: '#efafa9' });
```

Comment:
**suggestion**: Should we have an `AVOID` ring or are these set values that can't be changed?

Reply:
No, these are configurable. Not sure what the max number of rings is though. You're thinking of a `AVOID` in addition to `HOLD`. These rings are based on the example in backstage which was based on the Zolando one. https://opensource.zalando.com/tech-radar/ See the bottom of that page for definitions.